src/ripple/rpc/impl/Handler.cpp

```diff
@@ -47,6 +50,9 @@ template <class Object, class HandlerImpl>
 Status
 handle(JsonContext& context, Object& object)
 {
+    assert(
+        context.apiVersion >= HandlerImpl.minApiVer &&
```

Comment:
My compiler (apple-clang v15) is throwing this error on these two lines:
```
/Users/ckeshavabs/personal-code/rippled/src/ripple/rpc/impl/Handler.cpp:54:31: error: 'HandlerImpl' does not refer to a value
        context.apiVersion >= HandlerImpl.minApiVer &&
                              ^
/Users/ckeshavabs/personal-code/rippled/src/ripple/rpc/impl/Handler.cpp:49:31: note: declared here
template <class Object, class HandlerImpl>
                              ^
/Users/ckeshavabs/personal-code/rippled/src/ripple/rpc/impl/Handler.cpp:55:31: error: 'HandlerImpl' does not refer to a value
        context.apiVersion <= HandlerImpl.maxApiVer);
                              ^
/Users/ckeshavabs/personal-code/rippled/src/ripple/rpc/impl/Handler.cpp:49:31: note: declared here
template <class Object, class HandlerImpl>
```

Did you mean to use the scope-resolution operator? `HandlerImpl::minApiVer` -> this builds for me without a hitch

Reply:
you are right, thanks !